### PR TITLE
Add Review Status

### PR DESCRIPTION
**Pull request by meshde**
Review status can be one of:
* `queued`
* `completed`

Fix https://github.com/topcoder-platform/submissions-api/issues/130

### CODE REVIEW

File: src/services/ReviewService.js [hunk 1]
```diff
@@ -189,14 +191,15 @@ function * _updateReview (authUser, reviewId, entity) {
       'id': reviewId
     },
     UpdateExpression: `set score = :s, scoreCardId = :sc, submissionId = :su,
-                        typeId = :t, reviewerId = :r,
+                        typeId = :t, reviewerId = :r, status = :st
```
Comment (callmekatootie) [inline src/services/ReviewService.js]: @meshde Kindly add a comma here - I got a syntax error when making the api call:

```
{
    "message": "Invalid UpdateExpression: Syntax error; token: \"updated\", near: \":st\n                        updated =\""
}
```

Comment (callmekatootie) [reply src/services/ReviewService.js]: @meshde Also - `status` is a reserved keyword in dynamodb. You cannot use it directly. You will have to use `ExpressionAttributeNames`

Comment (meshde) [reply src/services/ReviewService.js]: @callmekatootie Have made the required fixes.

**Comment by callmekatootie** [comment]
@cwdcwd This PR is ready to be merged

**Pull request merged**